Makefile

```diff
@@ -385,11 +390,23 @@ proto:
 		--swagger_out=allow_merge=true:$(PWD)/proto-gen/openapi/ \
 		model/proto/api_v2/*.proto
 
+	$(PROTOC) \
+		$(PROTO_INCLUDES) \
+		-I plugin/storage/grpc/proto \
+				--gogo_out=plugins=grpc,$(PROTO_GOGO_MAPPINGS):$(PWD)/plugin/storage/grpc/proto/storage_v1 \
```

Comment:
in order to avoid code coverage drop, we should place the generated files under `proto-gen/storage_v1/`